common/beerocks/tlvf/yaml/beerocks/tlvf/beerocks_message_apmanager.yaml

```diff
@@ -25,6 +25,9 @@ cACTION_APMANAGER_4ADDR_STA_JOINED:
  
 cACTION_APMANAGER_JOINED_NOTIFICATION:
   _type: class
+  supported_channels:
```

Comment:
This should be in a separate commit with the autogenerated files changes (nitpick)